lib/jmap/core/method/request/query_method.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:jmap_dart_client/jmap/account_id.dart';
+import 'package:jmap_dart_client/jmap/core/method/method.dart';
+import 'package:jmap_dart_client/jmap/core/email/filter.dart';
+import 'package:jmap_dart_client/jmap/core/email/sort.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+abstract class QueryMethod extends MethodRequiringAccountId with OptionalFilter, OptionalSort {
```

Comment:
How about other arguments? `position` `anchor` `anchorOffset` `limit` `calculateTotal`?

Reply:
It is not a generic attribute. So will implement in specific queries

Reply:
srr, I can not get your point. please check the spec: https://jmap.io/spec-core.html#query